### PR TITLE
Error page styles

### PR DESCRIPTION
**Pull request by tomusdrw**
screenshot from 2016-08-31 15 38 00

### CODE REVIEW

File: home/web/client/error_pages.css [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+@font-face {
+    font-family: 'Roboto Mono';
+    src: url('~roboto-mono-webfont/fonts/RobotoMono-Light.ttf') format('truetype');
+    font-style: normal;
+    font-weight: 300;
+    text-rendering: optimizeLegibility;
+}
+
+html:root, :root body {
```
Comment (jacogr) [inline home/web/client/error_pages.css]: :root html ?

**Comment by jacogr** [comment]
Small issue, the rest looks good.

File: home/web/client/error_pages.css (hunk 1 above)
Comment (tomusdrw) [reply home/web/client/error_pages.css]: Wouldn't it be a no-op?
https://developer.mozilla.org/en-US/docs/Web/CSS/:root

Seems that `:root` is `html`, so there will be no descendant `html` inside `:root`.

Comment (jacogr) [reply home/web/client/error_pages.css]: So it should be 

```
:root, :root body {
```

Ok, didn't know that `:root` was basically `html` with a higher specificity, always have seen (and probably wrongly used) `:root html`

**Comment by tomusdrw** [comment]
Fixed.

**Pull request merged**